test/clj_docker_client/core_test.clj

```diff
@@ -120,6 +126,17 @@
         (rm conn id)))
     (image-rm conn img)))
 
+(deftest test-inspect
+  (with-open [conn ^DockerClient (connect)]
+    (pull conn img)
+    (testing "Inspecting a container"
+      (let [container-id (create conn img "echo ok" {} {})]
+        (start conn container-id)
+        (let [inspect-res (inspect conn container-id)
+              ip (-> inspect-res :network-settings :ip-address)]
+          (is (not (nil? ip)))
+          (is (correct-ip? ip)))))))
```

Comment:
Remove the container and the pulled image at the end of the test. See https://github.com/lispyclouds/clj-docker-client/blob/master/test/clj_docker_client/core_test.clj#L120-L121 Alternatively, can do this test in the `test-containers` test.

Reply:
Thanks, fixed!